website/docs/filter-group/add-filter/examples/add-filter.tsx

```diff
@@ -0,0 +1,223 @@
+        }}
+        onBlur={(e) => {
+          if (!value && !selectPopperRef.current?.contains(e.relatedTarget)) {
+            onClear();
+          }
+        }}
+        tag={FilterTrigger}
+        onClear={onClear}
+      >
+        {value}
+      </Select.Trigger>
+      <Select.Popper ref={selectPopperRef}>
+        <Select.List>
+          {options.map((option, idx) => (
+            <Select.Option value={option} key={idx}>
+              {option}
+            </Select.Option>
+          ))}
+        </Select.List>
+      </Select.Popper>
+    </Select>
+  );
+};
+
+type Filter = {
+  id: string;
+  name: string;
+  values: string[];
+};
+type FilterData = Record<string, string>;
+
+const Demo = () => {
+  const [selectedFilters, setSelectedFilters] = React.useState<Filter[]>([]);
+  const [filterData, setFilterData] = React.useState<FilterData>({});
+  const [addFilterVisible, setAddFilterVisible] = React.useState<boolean>(false);
+
+  const updateFilterFields = (filter: Filter) => {
+    setSelectedFilters((prevFilters) => [...prevFilters, filter]);
+  };
+
+  const updateFilterData = (value: string, name: string) => {
+    setFilterData((prevData) => {
+      return { ...prevData, ...{ [name]: value } };
+    });
+  };
+
+  const removeByName = (nameToRemove) => {
+    setSelectedFilters((prevFields) => {
+      return prevFields.filter(({ name }) => name !== nameToRemove);
+    });
+
+    setFilterData((prevData) => {
+      const newData = { ...prevData };
+      newData[nameToRemove];
+      return newData;
+    });
+  };
+
+  const clearAllFilters = () => {
+    setSelectedFilters([]);
+    setFilterData({});
+  };
+
+  const filtersWithoutSelected = React.useMemo(() => {
+    return filters.filter((filter) => {
+      return !selectedFilters.map((item) => item.name).includes(filter.name);
+    });
+  }, [filters, selectedFilters]);
+
+  const hasFilterData = React.useMemo(() => {
+    return Object.values(filterData).filter((v) => v).length > 0;
+  }, [filterData]);
+
+  return (
+    <Flex gap={2} flexWrap>
+      <Input w={160}>
+        <Input.Addon>
+          <SearchM />
+        </Input.Addon>
+        <Input.Value
+          value={filterData['name']}
+          onChange={(v) => updateFilterData(v, 'name')}
+          placeholder='Filter by name'
+          aria-label='Filter by name'
+        />
+        {filterData['name'] && (
+          <Input.Addon>
+            <Hint
+              tag={ButtonLink}
+              use='secondary'
+              addonLeft={CloseM}
+              title='Clear'
+              onClick={() => removeByName('name')}
+            />
+          </Input.Addon>
+        )}
+      </Input>
+
+      <Select value={filterData['size']} onChange={(v) => updateFilterData(v, 'size')}>
+        <Select.Trigger
+          empty={!filterData['size']}
+          placeholder='Size'
+          tag={FilterTrigger}
+          onClear={() => removeByName('size')}
+          aria-label='Select size'
```

Comment:
```suggestion
```
it's better not to add this aria-label, we already have the placeholder "Size" and role combobox, so SR reads everything that's needed ("size, combobox, collapsed"). So with this aria-label SR reads "size, select size..." And it can also be a problem for sighted voice control users, they see just "size" but voice control apps search for elements using their aria labels, so difference between visible text and aria label can be a problem for them

Reply:
Removed.